### DISCUSSION TITLE
Assistance Required: Confirmation of Alpha milestone Status

### DISCUSSION THREAD

**Discussion by dli459** [microsoft/SATechnicalOnboarding#129] [Questions]
Hello,

I have recently completed several learning paths aimed at achieving the Alpha milestone. Typically, I understand that confirmation emails and badges should be issued within 24 hours of completion. However, I have not received these acknowledgments on my MLSA profile yet. Furthermore, I reached out to [sa-help@microsoft.com] last week about this issue but have not yet received a response.

Could someone please assist me in reviewing if there was any error in my completion process, or if there are additional steps I need to undertake to finalize my status for the Alpha milestone?

**Comment by AnthonyByansi** [comment]
Hi @dli459 
Please note that Alpha milestone advancement is automated and can take 2-3 weeks from the time you completed the learning path.

**Reply by dli459** [reply]
Hi @AnthonyByansi,

Thank you for the update on the Alpha milestone advancement process. I appreciate the clarification and will wait for the automated update over the next few weeks.

Additionally, I am encountering an issue accessing the Student Ambassador handbook at this link: https://stdntpartners.sharepoint.com/_forms/default.aspx. Despite signing in with my Student Ambassador account, I am unable to open the document. Could you please assist me with this issue or direct me to someone who can help?

Thank you for your support.

**Reply by AnthonyByansi** [reply]
@dli459 Here is the correct handbook link.: https://stdntpartners.sharepoint.com/sites/SAProgramHandbook
Please let me know if you have any questions, I will be happy to help.

**Reply by dli459** [reply]
Hi Anthony,

Thank you for providing the updated link to the Student Ambassador handbook. Unfortunately, I'm still unable to access it; the link leads to an error message, which I've attached as the following image:
<img width="873" alt="handbook" src="https://github.com/microsoft/SATechnicalOnboarding/assets/55778381/99b8f503-a686-4aa1-bddf-d3de8548f1c0">

**Reply by AnthonyByansi** [reply]
@dli459 
It happens that you're trying to access the handbook through a different tenant. 
![handbook issue](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/0efde642-737b-4dd1-817a-2b4c2bc8a454)

Please open up a new Edge browser profile so that your signup process doesn't conflict with the existing profiles. Then sign in with your MLSA account

**Reply by AnthonyByansi** [reply]
@dli459 
Please follow these 2 steps to setup a new edge profile
![new profile](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/d96da2ba-8af7-4cc0-b2e1-a9c5231f2021)

**Reply by dli459** [reply]
Thank you for your suggestion.  I have tried accessing the handbook using a new Edge browser profile and signed in with my MLSA account.  However, I encountered an issue where the email suffix changes upon login. I've attached a screenshot showing the email suffix change for your reference.

Upon reflection, I suspect this issue might be because I am using my school email instead of a personal email.  Could this be causing the problem?  If so, how should we proceed to resolve it?  Should I change my MLSA email account, or is there another step I should take?
<img width="478" alt="Screen Shot 2024-04-29 at 18 29 11" src="https://github.com/microsoft/SATechnicalOnboarding/assets/55778381/7deffc29-25db-4c92-af19-e6d0443151f5">
<img width="550" alt="Screen Shot 2024-04-29 at 18 28 42" src="https://github.com/microsoft/SATechnicalOnboarding/assets/55778381/840876ae-e47c-458b-9a7f-b3a4707eec03">

**Reply by AnthonyByansi** [reply]
@dli459
To access the handbook, you must sign in using your MLSA account profile, not a personal or school email account. firstname.lastname@studentambassadors.com

**Reply by dli459** [reply]
The access issue has been resolved. Thank you for your assistance!

**Comment by AnthonyByansi** [comment]
> The access issue has been resolved. Thank you for your assistance!

Awesome. always happy to help!

**Reply by dli459** [reply]
Hi Anthony, I am writing to apologize and seek assistance with an issue concerning my Student Ambassadors Learn account.  I inadvertently linked my account to a Learn account created under a different email, neglecting the crucial information below:
<img width="1238" alt="Screen Shot 2024-05-18 at 12 19 48" src="https://github.com/microsoft/SATechnicalOnboarding/assets/55778381/6434ed01-5d9e-4e73-9f60-8d6b8d580912">

I have already disconnected the link between the two accounts. Could you please assist me in reconnecting my Student Ambassadors Learn account properly to ensure all activities are tracked correctly?   I am keen to fulfill the requirements and progress to the Alpha Student Ambassador milestone.

Thank you very much for your help and I apologize for any inconvenience caused.

**Reply by AnthonyByansi** [reply]
Hi @dli459 

I am unable to assist directly with reconnecting your Student Ambassadors Learn account, but our support team is well-equipped to help you with this issue and ensure all activities are tracked correctly. Please reach out to the SA-coordinator on teams to have you assisted.

Happy to help!

CC: @korriemsft @SalemTracey

**Reply by dli459** [reply]
Thanks Anthony, will wait for the reply.

**Comment by korriemsft** [comment]
Please contact ***@***.******@***.***> for assistance.
